templates/repo/issue/view_title.tmpl

```diff
@@ -47,7 +47,7 @@
 				{{if .HeadBranchLink}}
 					{{$headHref = printf "<a href=\"%s\">%s</a>" (.HeadBranchLink | Escape) $headHref}}
 				{{end}}
-				{{$headHref = printf "%s <a data-tooltip-content=\"%s\" data-clipboard-text=\"%s\">%s</a>" $headHref (.locale.Tr "copy_branch") (.HeadTarget | Escape) (svg "octicon-copy" 14)}}
+				{{$headHref = printf "%s <button class=\"button button-ghost\" data-tooltip-content=\"%s\" data-clipboard-text=\"%s\">%s</button>" $headHref (.locale.Tr "copy_branch") (.HeadTarget | Escape) (svg "octicon-copy" 14)}}
```

Comment:
Maybe  ``{{printf `%s then use quote " is fine`}}`` ? use `` ` `` for template string.

Reply:
Interesting, I didn't know go template can do backtick quotes. I tried single quotes before and it was rejected by the template compiler.